src/Models/FieldMaps/b1_dam.m

```diff
@@ -73,4 +73,17 @@
         end
         
     end
-end
+    
+    
+    methods(Access = protected)
```

Comment:
@agahkarakuzu could you review this PR? I think the changes in this file might conflict with the changes we're about to merge in your older B1dam PR (#290)